### PR TITLE
WIP: pythonPackages.django-storages: init at 1.7.1

### PR DESCRIPTION
**Pull request by peterromfeldhk**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/django-storages/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  propagatedBuildInputs = [ django ];
+
+  # django.core.exceptions.ImproperlyConfigured
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Provide a variety of Django storage backends";
+    homepage = https://github.com/jschneier/django-storages;
+    maintainers = with maintainers; [ Krajiyah ];
+    # license = with licenses; [ ??? ]; # not sure what license that is
```
Comment (worldofpeace) [inline pkgs/development/python-modules/django-storages/default.nix]: Also bsd3 clause

Comment (worldofpeace) [inline pkgs/development/python-modules/django-storages/default.nix]: Also 3-clause bsd.

File: pkgs/development/python-modules/django-storages/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchFromGitHub
```
Comment (worldofpeace) [inline pkgs/development/python-modules/django-storages/default.nix]:
```suggestion
{ stdenv, lib, buildPythonPackage, fetchFromGitHub
```

Comment (worldofpeace) [inline pkgs/development/python-modules/django-storages/default.nix]:
```suggestion
{ lib, buildPythonPackage, isPyPy, fetchFromGitHub
```

File: pkgs/development/python-modules/django-storages/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+, django }:
+
+buildPythonPackage rec {
+  pname = "django-storages";
+  version = "1.7.1";
+
+  disabled = isPyPy;
```
Comment (worldofpeace) [inline pkgs/development/python-modules/django-storages/default.nix]:
```suggestion
```

File: pkgs/development/python-modules/django-storages/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  src = fetchFromGitHub {
```
Comment (worldofpeace) [inline pkgs/development/python-modules/django-storages/default.nix]: Why github? The tarball on pypi looks fine after inspection.

**Comment by peterromfeldhk** [comment]
need to rethink django stuff

**Pull request closed without merging**